pulp-dev.py

```diff
@@ -104,6 +104,7 @@
         # Lazy Streamer components
         ('server/srv/pulp/streamer.tac', '/srv/pulp/streamer.tac'),
         ('server/srv/pulp/streamer_auth.wsgi', '/srv/pulp/streamer_auth.wsgi'),
+        ('server/etc/httpd/conf.d/pulp_lazy.conf', '/etc/httpd/conf.d/pulp_lazy.conf'),
```

Comment:
I think `pulp_streamer.conf` would be more appropriate.  The term `lazy` is a broader term for the general concept which includes more than the streamer.